src/js/button.js

```diff
@@ -27,64 +24,65 @@ var Button = Component.extend({
     this.on('focus', this.onFocus);
     this.on('blur', this.onBlur);
   }
-});
-
-Component.registerComponent('Button', Button);
 
-Button.prototype.createEl = function(type, props){
-  // Add standard Aria and Tabindex info
-  props = Lib.obj.merge({
-    className: this.buildCSSClass(),
-    'role': 'button',
-    'aria-live': 'polite', // let the screen reader user know that the text of the button may change
-    tabIndex: 0
-  }, props);
-
-  let el = Component.prototype.createEl.call(this, type, props);
-
-  // if innerHTML hasn't been overridden (bigPlayButton), add content elements
-  if (!props.innerHTML) {
-    this.contentEl_ = Lib.createEl('div', {
-      className: 'vjs-control-content'
-    });
-
-    this.controlText_ = Lib.createEl('span', {
-      className: 'vjs-control-text',
-      innerHTML: this.localize(this.buttonText) || 'Need Text'
-    });
-
-    this.contentEl_.appendChild(this.controlText_);
-    el.appendChild(this.contentEl_);
+  createEl(type, props) {
+    // Add standard Aria and Tabindex info
+    props = Lib.obj.merge({
+      className: this.buildCSSClass(),
+      'role': 'button',
+      'aria-live': 'polite', // let the screen reader user know that the text of the button may change
+      tabIndex: 0
+    }, props);
+
+    let el = Component.prototype.createEl.call(this, type, props);
+
+    // if innerHTML hasn't been overridden (bigPlayButton), add content elements
+    if (!props.innerHTML) {
+      this.contentEl_ = Lib.createEl('div', {
+        className: 'vjs-control-content'
+      });
+
+      this.controlText_ = Lib.createEl('span', {
+        className: 'vjs-control-text',
+        innerHTML: this.localize(this.buttonText) || 'Need Text'
+      });
+
+      this.contentEl_.appendChild(this.controlText_);
+      el.appendChild(this.contentEl_);
+    }
+
+    return el;
   }
 
-  return el;
-};
-
-Button.prototype.buildCSSClass = function(){
-  // TODO: Change vjs-control to vjs-button?
-  return 'vjs-control ' + Component.prototype.buildCSSClass.call(this);
-};
+  buildCSSClass() {
+    // TODO: Change vjs-control to vjs-button?
+    return 'vjs-control ' + Component.prototype.buildCSSClass.call(this);
+  }
 
   // Click - Override with specific functionality for button
-Button.prototype.onClick = function(){};
+  onClick(){}
 
   // Focus - Add keyboard functionality to element
-Button.prototype.onFocus = function(){
-  Events.on(document, 'keydown', Lib.bind(this, this.onKeyPress));
-};
+  onFocus(){
```

Comment:
Yep, spacing. It would be interesting to have a pre-commit hook that modifying the code styles based on output from node-jscs and jshint/eslint.
Also, I'm still debating whether we should have a space before `()` in the concise methods.
i.e.
```js
{
  foo() {}
}
// vs
{
  foo () {}
}
```

For function statements, we're supposed to put a space after the name and this sets it apart from where we call the function and where we declare the function. The same could be applied to the concise methods.